docs/UserGuide.adoc

```diff
@@ -75,6 +75,41 @@ You can use `entrylist` and `orderlist` to switch between the two lists.
 
 Person List Panel, Center Panel and Right Panel will be explained in greater detail in <<Features>>.
 
+=== Walkthrough
+Here's a simple walkthrough of Contact Sails.
```

Comment:
This is a good section, but the rest of us have given walkthroughs in the Features Section. Perhaps you can shift this to the respective command sections?

Reply:
To my understanding, walkthrough in features section are for features. This is a general walkthrough at the start, before the features section.